### PR TITLE
net-firewall/ferm: version bump to 2.3

### PR DESCRIPTION
**Pull request by hydrapolic**
Package-Manager: portage-2.3.0_rc1

### CODE REVIEW

**Comment by Amynka** [comment]
@gentoo/proxy-maint

File: net-firewall/ferm/ferm-2.3.ebuild [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+EAPI=6
+
+inherit eutils systemd
+
+DESCRIPTION="Command line util for managing firewall rules"
+HOMEPAGE="http://ferm.foo-projects.org/"
+SRC_URI="http://ferm.foo-projects.org/download/${PV}/${P}.tar.gz"
+
+LICENSE="GPL-2"
```
Comment (gktrk) [inline net-firewall/ferm/ferm-2.3.ebuild]: This should be [GPL-2+](https://github.com/MaxKellermann/ferm/blob/master/src/ferm#L16) I think

File: net-firewall/ferm/ferm-2.3.ebuild [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+src_install() {
+	dosbin src/{,import-}ferm
+	systemd_dounit ferm.service
+
+	einstalldocs
+	doman doc/*.1
+}
+
+pkg_postinst() {
+	elog "See /usr/share/doc/${PF}/examples for sample configs"
```
Comment (gktrk) [inline net-firewall/ferm/ferm-2.3.ebuild]: This should be "See ${EROOT}usr/share ..." I think

File: net-firewall/ferm/ferm-2.3.ebuild [hunk 3]
```diff
@@ -0,0 +1,38 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+inherit eutils systemd
```
Comment (gktrk) [inline net-firewall/ferm/ferm-2.3.ebuild]: I don't think any of the functionality of eutils is being utilized, is it necessary?

File: net-firewall/ferm/ferm-2.3.ebuild (hunk 1 above)
Comment (hydrapolic) [inline net-firewall/ferm/ferm-2.3.ebuild]: On the official page they have: Licensed under the GPLv2. They have a link to the license: http://ferm.foo-projects.org/download/COPYING. Licenses are not my field of expertise, if you say so I can change it.

File: net-firewall/ferm/ferm-2.3.ebuild (hunk 3 above)
Comment (hydrapolic) [inline net-firewall/ferm/ferm-2.3.ebuild]: The only thing used is einstalldocs.

Comment (gktrk) [inline net-firewall/ferm/ferm-2.3.ebuild]: einstalldocs is an EAPI provided helper. See https://dev.gentoo.org/~ulm/pms/head/pms.html#x1-145034r25. you can safely remove eutils if that's the only assumed use of it.

File: net-firewall/ferm/ferm-2.3.ebuild (hunk 1 above)
Comment (gktrk) [inline net-firewall/ferm/ferm-2.3.ebuild]: The license can be GPLv2-only or GPLv2+. COPYING wouldn't tell you that. It's better to check the source files themselves for this.

**Pull request closed without merging**